pkg/templatelookup/modulestatusinfo.go

```diff
@@ -13,52 +13,52 @@ var (
 	ErrInvalidModuleInStatus = errors.New("invalid module entry in Kyma status")
 )
 
-type AvailableModule struct {
+type ModuleStatusInfo struct {
 	v1beta2.Module
-	Enabled         bool
+	IsEnabled       bool
 	ValidationError error
-	Unmanaged       bool
+	IsUnmanaged     bool
 }
 
-func (a AvailableModule) IsInstalledByVersion() bool {
+func (a ModuleStatusInfo) IsInstalledByVersion() bool {
 	return a.configuredWithVersionInSpec() || a.installedwithVersionInStatus()
 }
 
 // configuredWithVersionInSpec returns true if the Module is enabled in Spec using a specific version instead of a channel.
-func (a AvailableModule) configuredWithVersionInSpec() bool {
-	return a.Enabled && a.Version != "" && a.Channel == ""
+func (a ModuleStatusInfo) configuredWithVersionInSpec() bool {
+	return a.IsEnabled && a.Version != "" && a.Channel == ""
 }
 
 // installedwithVersionInStatus returns true if the Module installed using a specific version (instead of a channel) is reported in Status.
-func (a AvailableModule) installedwithVersionInStatus() bool {
-	return !a.Enabled && shared.NoneChannel.Equals(a.Channel) && a.Version != ""
+func (a ModuleStatusInfo) installedwithVersionInStatus() bool {
+	return !a.IsEnabled && shared.NoneChannel.Equals(a.Channel) && a.Version != ""
 }
 
-// FindAvailableModules returns a list of AvailableModule objects based on the Kyma CR Spec and Status.
-func FindAvailableModules(kyma *v1beta2.Kyma) []AvailableModule {
+// FetchModuleStatusInfo returns a list of ModuleStatusInfo objects based on the Kyma CR Spec and Status.
+func FetchModuleStatusInfo(kyma *v1beta2.Kyma) []ModuleStatusInfo {
```

Comment:
But this is not all coming from module status, it contains the information from kyma.spec.modules

Reply:
I see the point that `ModuleStatusInfo` may be confused with the `ModuleTemplate.status` sub-resource. How about we just name it `ModuleInfo` instead? Specific enough to say that it is information about a module, but generic enough so that we can place all information we want in it?